engine/evd.mli

```diff
@@ -28,6 +28,9 @@ open Environ
     It also contains conversion constraints, debugging information and
     information about meta variables. *)
 
+type econstr
+type etypes = econstr
```

Comment:
Why are these signatures different from those in [evd.ml](https://github.com/coq/coq/pull/7136/files#diff-d5eab4d1ae33866156756e9ba32f0f2eR24)?
> type econstr = constr
type etypes = types